tests/sentry/incidents/endpoints/test_project_alert_rule_details.py

```diff
@@ -424,15 +389,124 @@ def test_no_owner(self):
         assert resp.data == serialize(alert_rule, self.user)
         assert resp.data["owner"] is None
 
+    @responses.activate
+    def test_success_response_from_sentry_app(self):
```

Comment:
Tests that endpoint properly handles metric alerts with Sentry App actions (PUT)